releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -6,6 +6,11 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
+## 1.2.7 - July 14, 2020
+
+* Fixed the bug where setting `spring.cloud.azure.appconfiguration.enabled` to false while using `spring-cloud-azure-appconfiguration-config-web`, resulted in an error on startup. [#332](https://github.com/Azure/AppConfiguration/issues/332)
+* Fixed issues where a 400 Error would be returned from App Configuration when multiple labels and profiles are used at the same time. [#675](https://github.com/microsoft/spring-cloud-azure/issues/675)
```

Comment:
It's "issues" or just one issue? Should we follow the style of others and say "Fixed the bug..."?

Reply:
issues as it could happen at startup or refresh for using many labels or profiles resulting in 6 or more results.

Reply:
But it's the same cause (bug), right?